src/V3EmitC.cpp

```diff
@@ -3901,6 +4060,7 @@ class EmitCTrace final : EmitCStmts {
                 putsDecoration("// Final\n");
                 iterateAndNextNull(nodep->finalsp());
             }
+            m_useSelfForThis = false;
```

Comment:
Remove and use a VL_RESTORER earlier instead

Reply:
Fancy writing nested function definitions? :) Ok, will do.